haiku-apps/desknotes/desknotes-1.1.recipe

```diff
@@ -10,7 +9,8 @@ HOMEPAGE="https://github.com/HaikuArchives/DeskNotes"
 COPYRIGHT="2000 Colin Stewart"
 LICENSE="BSD (3-clause)"
 REVISION="4"
-SOURCE_URI="$(HOMEPAGE)/archive/e39a1cf592c5a51f49578dd5efe0bbb1c6d0f450.tar.gz"
+SOURCE_URI="https://github.com/HaikuArchives/DeskNotes/archive/e39a1cf592c5a51f49578dd5efe0bbb1c6d0f450.tar.gz"
+SOURCE_DIR="DeskNotes-e39a1cf592c5a51f49578dd5efe0bbb1c6d0f450"
```

Comment:
I would recommend replacing
```
SOURCE_URI="$(HOMEPAGE)/archive/e39a1cf592c5a51f49578dd5efe0bbb1c6d0f450.tar.gz"
```
by:
```
srcGitRev="e39a1cf592c5a51f49578dd5efe0bbb1c6d0f450"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
SOURCE_DIR="DeskNotes-$srcGitRev"
```
BTW, you forgot to add the CHECKSUM_SHA256.